gobblin-core/src/main/java/gobblin/converter/jdbc/JdbcEntryDatum.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.converter.jdbc;
+
+import com.google.common.base.Preconditions;
+
+
+public class JdbcEntryDatum {
+  private final String columnName;
+  private final Object val;
+  private int byteSize;
+
+  public JdbcEntryDatum(String columnName, Object val) {
+    this.columnName = Preconditions.checkNotNull(columnName);
+    this.val = Preconditions.checkNotNull(val);
+    this.byteSize = val.toString().getBytes().length;
```

Comment:
The main purpose of this is to control buffer size of buffered insert and it's better than having batch size to limit buffer size as each row size is different.
This cannot be 100% accurate as each JVM has its own way to store String and I think it can be used as an measurement. (Performance can be benefit by not having this, but, for now, I don't see any performance issue with the performance test.)